pom.xml

```diff
@@ -740,6 +704,7 @@
             <modules>
                 <module>delta-lake/delta-stub</module>
                 <module>iceberg</module>
+                <module>iceberg-common</module>
```

Comment:
We are missing iceberg in the 355 profile. Does Iceberg 1.6.1 work with Spark 3.5.5?